frontend/src/app/events/event-routing.module.ts

```diff
@@ -10,7 +10,7 @@ import { AuthGuard } from '../auth/guards/auth/auth.guard';
 const routes: Routes = [
   { path: '', component: FeedPage, canActivate: [ AuthGuard ], pathMatch: 'full' },
   { path: 'create', component: EventCreatePage, canActivate: [ AuthGuard ] },
-  { path: ':id', component: EventPage, canActivate: [ AuthGuard ] },
+  { path: ':id', component: EventPage },
```

Comment:
everybody should be able to any events. We have a requirement for that : remember guest users shall be able to see public content